crates/db_views_actor/src/person_view.rs

```diff
@@ -100,6 +100,7 @@ impl<'a> PersonQuery<'a> {
       SortType::Hot | SortType::Active | SortType::TopAll => {
         query.order_by(person_aggregates::comment_score.desc())
       }
+      SortType::Controversial => query.order_by(person_aggregates::comment_score.asc()),
```

Comment:
This doesn't look right to me, should we maintain a `person_aggregates::controversy_score`?

Reply:
In what sense is a person controversial though? I find that idea pretty controversial itself :P 

I just added that as a dumb default since there probably won't be such an option ever implemented in the UI, I could also put it along with SortType::Hot even if it's quite a different concept.

Reply:
Ah right, so this was added because the same `SortType` is used here both for posts and for persons, that seems less than ideal. You could add it to the match pattern above together with hot, active, and top all, which also don't really apply to a person.

Might be better though to split that out, so that there's a `PostSortType` and `PersonSortType`, alongside the existing `CommentSortType`. That's a bigger change though 🙂 WDYT?

Reply:
I agree it would make more sense to add a separate sort type for person, and then also get rid of the unused options (newcomments, hot, active)

Reply:
Yeah that would be quite better

Reply:
I made PersonSortType like this:

```rust
pub enum PersonSortType {
  New,
  Old,
  MostComments,
  TopDay,
  TopWeek,
  TopMonth,
  TopYear,
  TopAll,
}
```

Do we need the same with CommunitySortType? If so, what values should be removed from it?